app/src/main/java/org/wikipedia/feed/mainpage/MainPageCardView.java

```diff
@@ -0,0 +1,58 @@
+package org.wikipedia.feed.mainpage;
+
+import android.content.Context;
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import org.wikipedia.R;
+import org.wikipedia.feed.view.CardFooterView;
+import org.wikipedia.feed.view.CardHeaderView;
+import org.wikipedia.feed.view.DefaultFeedCardView;
+import org.wikipedia.feed.view.FeedAdapter;
+import org.wikipedia.history.HistoryEntry;
+import org.wikipedia.page.PageTitle;
+import org.wikipedia.settings.SiteInfoClient;
+import org.wikipedia.util.L10nUtil;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class MainPageCardView extends DefaultFeedCardView<MainPageCard> {
+    @BindView(R.id.view_static_card_container) View containerView;
+    @BindView(R.id.view_static_card_header) CardHeaderView headerView;
+    @BindView(R.id.view_static_card_footer) CardFooterView footerView;
+
+    public MainPageCardView(@NonNull Context context) {
+        super(context);
+        inflate(getContext(), R.layout.view_static_card, this);
+        ButterKnife.bind(this);
+    }
+
+    @Override public void setCard(@NonNull final MainPageCard card) {
+        super.setCard(card);
+        setLayoutDirectionByWikiSite(card.wikiSite(), containerView);
+
+        headerView.setTitle(L10nUtil.getStringForArticleLanguage(getCard().wikiSite().languageCode(), R.string.view_main_page_card_title))
+                .setLangCode(getCard().wikiSite().languageCode())
+                .setCard(getCard())
+                .setCallback(getCallback());
+
+        footerView.setCallback(this::goToMainPage);
+        footerView.setFooterActionText(L10nUtil.getStringForArticleLanguage(getCard().wikiSite().languageCode(), R.string.view_main_page_card_action));
```

Comment:
In other cards, we don't use `L10nUtil.getStringForArticleLanguage()` on the footer action text. Not sure if we are going to do this here.

Reply:
Agreed, let's keep it consistent with other cards.